app/Http/Controllers/ViewAssetsController.php

```diff
@@ -361,19 +377,23 @@ public function postAcceptAsset($logID = null)
                 ->update(array('assigned_to' => null));
             }
         }
-        $logaction->target_id = $findlog->target_id;
 
+        $logaction->target_id = $findlog->target_id;
         $logaction->note = e(Input::get('note'));
-        $logaction->user_id = $user->id;
```

Comment:
Was this an intentional removal of the user ID?  Hard to tell from this context, just want to make sure.